xonsh/tools.py

```diff
@@ -428,16 +428,59 @@ def print_exception(msg=None):
     if 'XONSH_SHOW_TRACEBACK' not in env:
         sys.stderr.write('xonsh: For full traceback set: '
                          '$XONSH_SHOW_TRACEBACK = True\n')
```

Comment:
Should we also expand this message to mention `$XONSH_TRACEBACK_LOGFILE`, or no?